Header header 
int32[] comm_latency        
